apps/client/src/components/toast/toast.tsx

```diff
@@ -1,19 +1,40 @@
+import { useState } from 'react';
 import styles from './toast.module.css';
+import { ToastEdit } from '../';
+import EditIcon from '@mui/icons-material/Edit';
 
 type ToastProps = {
-  user: string;
+  toastId: string;
+  username: string;
   date: string;
+  hasHappened: boolean;
 };
 
-export const Toast = (props: ToastProps) => {
-  if (props.user === '') {
+export const Toast: React.FC<ToastProps> = ({
+  username,
+  date,
+  toastId,
+  hasHappened,
+}) => {
+  const [editToastModal, setEditToastModal] = useState(false);

```

Comment:
I want the name of this state to be move booleany
for example `isEditToastModalOpen` or simply `isOpen` 
Something that sounds like a question.